### DISCUSSION TITLE
Led ring and flow deck v2

### DISCUSSION THREAD

**Discussion by minpark0120** [bitcraze/discussions#878] [Q&A]
hi, I'm working on a code that changes the led ring at the same time that the crazyflie cluster flies, but the problem is that the led ring doesn't light up when I connect both flow deck v2 and led ring. As a solution to this problem, I tried to connect only gnd, vcom, i02, and ow by wire, but the light didn't come on.  Is there any solutions to connect both?

**Comment by tobbeanton** [comment]
Hmm, it should work, at least it has been working. You can get some info from [this](https://www.hackster.io/krichardsson/light-paint-with-a-drone-d050af) pretty old hackster project.

What does the [console](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/userguide_client/console_tab/) output in the cfclient print?
